src/components/GroceryItems.vue

```diff
@@ -11,11 +11,7 @@
           <span
             class="letter"
             :class="isTyping && 'letter-active'"
-            :style="
-              i % getRandomIntFrom(2) === 0 && {
-                filter: `drop-shadow(0 0 ${getRandomIntFrom(8)}px ${colors[i]})`
-              }
-            "
+            :style="letterStyle(i)"
```

Comment:
I think this one makes it even neater and this way we don't have to pass the function to become a method.